src/dialog/marisa.c

```diff
@@ -801,20 +801,19 @@ DIALOG_TASK(marisa, StageExPreBoss) {
 	MSG(marisa, "Goofin’ off and thinkin’ about weird stuff keeps your mind sharp! Let’s you recharge!");
 
 	FACE(yumemi, smug);
-	MSG(yumemi, "Yes, of course it’d work for the likes of you.");
+	MSG(yumemi, "How nice for you. Too bad you're not real.");
 	FACE(yumemi, normal);
-	MSG(yumemi, "But for us mere humans, well, we’re unable to handle the Siren’s song of these places.");
 
 	FACE(marisa, normal);
 	MSG(marisa, "I really am human, y’know.");
```

Comment:
Doesn't seem to fit as well without that deleted line
```diff
-	MSG(yumemi, "But for us mere humans, well, we’re unable to handle the Siren’s song of these places.");
```

Reply:
How do you l like the new line?

Reply:
Maybe the connection to the next line would still be just a tiny bit more clear if she said "humans" instead of "mortals", but it's fine either way, really. Keep the version you prefer. As for me, I'd say the new line works just about as well as the old one. I did like the Siren's song metaphor, though.